### PR TITLE
Minor docs/stubs improvements in `invlerp`/`remap`

### PR DESCRIPTION
**Pull request by ankith26**
Just a follow up PR to #2654

The main goal of doing this PR was to fix the `ZeroDivisionError` mentioned in the docs to `ValueError`, which is what is actually raised in the code. Also made one argument name in stubs consistent with the docs.

While I was editing the docs, I also polished it up a bit up according to my tastes.

### CODE REVIEW

**Review by Damus666 — APPROVED** [review]
Indeed clearer, nice

**Review by MyreMylar — APPROVED** [review]
Yep, looks nicer I agree. I think `lerp()` has the same ambiguity between weight/value.

File: buildconfig/stubs/pygame/math.pyi [hunk 1]
```diff
@@ -334,7 +334,7 @@ class Vector3(_GenericVector):
 
 
 def lerp(a: float, b: float, weight: float, do_clamp: bool = True, /) -> float: ...
```
Comment (MyreMylar) [inline buildconfig/stubs/pygame/math.pyi:336]:
```suggestion
def lerp(a: float, b: float, value: float, do_clamp: bool = True, /) -> float: ...
```

**Review by Starbuck5 — APPROVED** [review]
OK

**Pull request merged**